src/api/xxAMIDOxx.xxSTACKSxx.API/Authentication/SwaggerGenOptionsExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace xxAMIDOxx.xxSTACKSxx.API.Authentication
+{
+    public static class SwaggerGenOptionsExtensions
+    {
+        public static void ConfigureForJwtBearerAuthentication(
+            this SwaggerGenOptions options,
+            JwtBearerAuthenticationConfiguration jwtBearerAuthenticationConfiguration)
+        {
+            if (!jwtBearerAuthenticationConfiguration.HasOpenApiClient())
+            {
+                return;
+            }
+
+            options.AddSecurityDefinition(OpenApiSecurityDefinitions.OAuth2, new OpenApiSecurityScheme
+            {
+                Type = SecuritySchemeType.OAuth2,
+                Flows = new OpenApiOAuthFlows
+                {
+                    // Recommended flow (Authorization Code with PKCE - https://tools.ietf.org/html/draft-ietf-oauth-browser-based-apps-00)
+                    AuthorizationCode = new OpenApiOAuthFlow
```

Comment:
as mentioned in the config, you should probably use the `openId-configuration` endpoint to load these and just pass the configuration endpoint to the settings

Reply:
See my earlier comment